src/Captioning/Format/WebvttFile.php

```diff
@@ -179,4 +174,39 @@ public function buildPart($_from, $_to)
 
         return $this;
     }
+
+    /**
+     * @return array
+     * @throws \Exception
+     */
+    private function getFileContentAsArray()
```

Comment:
I'd rather move those two methods up to the abstract File class.
It would prevent duplicate code and may be useful to other format as well.